### PR TITLE
CTOR-1050 : Refresh cloud-aws-backup

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-aws-backup

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md [hunk 1]
```diff
@@ -5,42 +5,60 @@ title: Amazon Backup Vault
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-backup.md [hunk 2]
```diff
@@ -5,42 +5,59 @@ title: Amazon Backup Vault
...
+</TabItem>
+</Tabs>
 
+### Discovery rules
+
+#### Host discovery
+
+| Rule name                       | Description               |
+|:--------------------------------|:--------------------------|
+| Amazon Web Service Backup Vault | Discover AWS Backup Vault |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-backup.md:42]:
```suggestion
| Amazon Web Service Backup Vault | Discover AWS Backup Vaults |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md [hunk 3]
```diff
@@ -81,109 +96,292 @@ sudo ./aws/install
...
+que le connecteur arrive bien à superviser une instance AWS en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_aws_backup_api.pl \
+	--plugin=cloud::aws::backup::plugin \
+	--mode=job-status \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md:253]: ?

File: pp/integrations/plugin-packs/procedures/cloud-aws-backup.md [hunk 4]
```diff
@@ -81,107 +95,292 @@ sudo ./aws/install
...
-/usr/lib/centreon/plugins//centreon_aws_backup_api.pl \
-    --plugin=cloud::aws::backup::plugin \
-    --mode=jobstatus \
-    --help
+/usr/lib/centreon/plugins/centreon_aws_backup_api.pl \
+	--plugin=cloud::aws::backup::plugin \
+	--mode=job-status \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-backup.md:252]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md [hunk 5]
```diff
@@ -81,109 +96,292 @@ sudo ./aws/install
...
+| AWSACCESSKEY    | Set AWS access key                                                                                                                                 |                   |             |
+| AWSASSUMEROLE   | Set Amazon Resource Name of the role to be assumed                                                                                                 |                   |             |
+| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option                         | awscli            |             |
+| AWSREGION       | Set the region name (required)                                                                                                                     |                   | X           |
+| AWSSECRETKEY    | Set AWS secret key                                                                                                                                 |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md:200]:
```suggestion
| AWSACCESSKEY    | Set AWS access key                                                                                                                                 |                   | X           |
| AWSASSUMEROLE   | Set Amazon Resource Name of the role to be assumed                                                                                                 |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option                         | awscli            |             |
| AWSREGION       | Set the region name (required)                                                                                                                     |                   | X           |
| AWSSECRETKEY    | Set AWS secret key                                                                                                                                 |                   | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md (hunk 3 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-backup.md:253]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-backup.md [hunk 6]
```diff
@@ -81,107 +95,292 @@ sudo ./aws/install
...
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AWSACCESSKEY    | Set AWS access key                                                                                                                       |                   |             |
+| AWSASSUMEROLE   | Set Amazon Resource Name of the role to be assumed                                                                                       |                   |             |
+| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option               | awscli            |             |
+| AWSREGION       | Set the region name (required)                                                                                                           |                   | X           |
+| AWSSECRETKEY    | Set AWS secret key                                                                                                                       |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-backup.md:201]:
```suggestion
| AWSACCESSKEY    | Set AWS access key                                                                                                                       |                   | X           |
| AWSASSUMEROLE   | Set Amazon Resource Name of the role to be assumed                                                                                       |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option               | awscli            |             |
| AWSREGION       | Set the region name (required)                                                                                                           |                   | X           |
| AWSSECRETKEY    | Set AWS secret key                                                                                                                       |                   | X           |
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-backup.md (hunk 4 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-backup.md:252]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**